library/src/main/java/com/chuckerteam/chucker/internal/support/ShareUtil.kt

```diff
@@ -0,0 +1,41 @@
+package com.chuckerteam.chucker.internal.support
+
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.app.ShareCompat
+import com.chuckerteam.chucker.R
+
+fun AppCompatActivity.shareError(errorText: String) {
```

Comment:
I'm a bit against those extensions function. I would prefer to have those two as methods inside the `TransactionActivity` and `ErrorActivity`. First those two functions are public and will be exposed to users of Chucker (that is nothing we would love to do).

Moreover I'd try to the keep all the UI related code (e.g. `getString(R.string.chucker_share_error_title)`) closer to the Activity rather than inside a Util class.

Reply:
Yeah, my mistake about exposing.
As to keep `R` class - agree as well. 

Will reiterate on this one.